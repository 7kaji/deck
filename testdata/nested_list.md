# Nested List

---

# 4-space padding

- A
    - B
        - C

---

# 2-space padding

- A
  - B
    - C
