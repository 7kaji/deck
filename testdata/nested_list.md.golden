[
  {
    "layout": "",
    "titles": [
      "Nested List"
    ]
  },
  {
    "layout": "",
    "titles": [
      "4-space padding"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "B"
              }
            ],
            "bullet": "-",
            "nesting": 1
          },
          {
            "fragments": [
              {
                "value": "C"
              }
            ],
            "bullet": "-",
            "nesting": 2
          }
        ]
      }
    ]
  },
  {
    "layout": "",
    "titles": [
      "2-space padding"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "B"
              }
            ],
            "bullet": "-",
            "nesting": 1
          },
          {
            "fragments": [
              {
                "value": "C"
              }
            ],
            "bullet": "-",
            "nesting": 2
          }
        ]
      }
    ]
  }
]